### ISSUE TITLE
Theano is not multithreading???

### ISSUE THREAD

**Issue by akalenda** [akalenda/PyFmask#13]
I was under the impression that Theano would use multiple cores, but that does not seem to be the case. Fixing this is a high priority, because it will drastically reduce the time it takes to test code.

It may be that I do not have Theano configured correctly. This is the first thing to investigate. If Theano requires special configuration to take advantage of multiple cores, we need to document it in our own readme.

Failing that, it's time to look at alternatives to Theano. TensorFlow, maybe? Or that thing Microsoft has, uhhh, it's called CTNK or something.

**Comment by akalenda** [comment]
PyTorch looks promising. Very similar to Theano in its styling, but purports to work with the graphics card as well.
